src/main/java/org/onebusaway/alexa/AnonSpeechlet.java

```diff
@@ -261,4 +266,23 @@ private SpeechletResponse askForCity(Optional<String> currentCityName) {
         }
         return SpeechletResponse.newAskResponse(out, cityReprompt);
     }
+
+    private String allRegionsSpoken() {
+        List<String> activeRegions = obaClient.getAllRegions()
+                .stream()
+                .filter(r -> RegionUtils.isRegionUsable(r) && !r.getExperimental())
```

Comment:
Agreed, I was thinking the same thing.  This was originally created as more of a logical discovery check based on the attributes of the OBA server, not the presence or absence of fields - it was also assumed that obaBaseUrl was required.  Just opened https://github.com/OneBusAway/onebusaway-client-library/issues/8 to update the library for this.